README.md

```diff
@@ -36,7 +36,19 @@ Python package to develop applications with the Dispatch platform.
 
 ## What is Dispatch?
 
-Dispatch is a platform for developing scalable & reliable distributed systems.
+Dispatch is a cloud service for developing scalable and reliable applications in
+Python, including:
+
+- **Event-Driven Architectures**
+- **Background Jobs**
+- **Transactional Workflows**
+- **Multi-Tenant Data Pipelines**
+
+Dispatch differs from alternative solutions by allowing developers top write
+simple Python code as if it was a local application: it has a **minimal API
```

Comment:
Local application?

Reply:
what I mean is you don't have to bother with defining what how things get distributed and where, it's code that looks like it could run on a local instance of the program, but Dispatch takes care of scheduling across nodes and state management.

Reply:
I changed the wording to remove this part since it's confusing.